lib/inc/cpp-pcp-client/util/logging.hpp

```diff
@@ -16,18 +17,31 @@ namespace leatherman {
    within the cpp-pcp-client library for logging to work.
 */
 
+#define LOG_ACCESS(message) PCPClient::Util::logAccess(message, __LINE__);
+
 namespace PCPClient {
 namespace Util {
 
+void doLogAccess(std::string const& message,
+                 int line_num);
+
+static inline void logAccess(std::string const& message,
+                             int line_num)
+{
+    doLogAccess(message, line_num);
```

Comment:
I kept the same leatherman's structure and initially it was more complex.
Will remove that.